app/controllers/patients_controller.rb

```diff
@@ -56,6 +56,17 @@ def data_entry_create # temporary
       render 'data_entry'
     end
   end
+
+
+  def fetch_clinic_naf
+    accepts_naf =  Clinic.find_by(id: params['id']).accepts_naf
+    render json: [params['id'], accepts_naf]
+  end
+
+  def get_clinics
```

Comment:
Same deal -- lets move this to the clinics controller. Might make a good thing to return from the index route if you're requesting JSON?